### PR TITLE
Initial Dutch translation files

### PR DESCRIPTION
**Pull request by roru69**

### CODE REVIEW

**Comment by marius-wieschollek** [comment]
Is `nl_NL` actually used by Nextcloud? If I set the language to "Nederlands", the `nl` files are used.

File: src/l10n/nl.js [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+            'Overwrite existing'                  : 'Bestaande overschrijven',
+            'Merge with existing'                 : 'Samenvoegen met bestaande',
+            'Create new entry'                    : 'Nieuw record aanmaken',
+            'For encrypted backups'               : 'Verplicht bij versleutelde backups',
+            'CSV Options'                         : 'CSV Opties',
+            'Database'                            : 'Database',
+            'Quote Character'                     : 'Aanhalingsteken',
+            'Escape Character'                    : 'Escape karakter',
+            'Field Delimiter'                     : 'Veldbegrenzer',
+            'Quote'                               : 'Hochkomma',
```
Comment (marius-wieschollek) [inline src/l10n/nl.js]: Is "Hochkomma" correct as translation for a double quote?

File: src/l10n/nl.js [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+            'Include special characters'          : 'Bijzondere tekens gebruiken',
+            'User Interface'                      : 'Gebruikersinterface',
+            'Initial section'                     : 'Startpagina',
+            'All Passwords'                       : 'Alle wachtwoorden',
+            'Passwords List View'                 : 'Wachtwoorden in lijstweergave',
+            'Set title from'                      : 'Dit veld als titel gebruiken',
+            'Sort by'                             : 'Sorteer op',
+            'Title field'                         : 'Titelveld',
+            'Single click action'                 : 'Atie bij klik',
+            'Double click action'                 : 'Aktie bij dubbelklik',
```
Comment (marius-wieschollek) [inline src/l10n/nl.js]: In line 249 its "Atie", please confirm

File: src/l10n/nl.json [hunk 3]
```diff
@@ -0,0 +1,115 @@
+{
+    "translations": {
+        "Passwords"                                                                                 : "Passwords",
```
[the remark this answers is not in the document]
Comment (roru69) [reply src/l10n/nl.json]: Made the dutch translation. Corrected the issue.

File: src/l10n/nl.js [hunk 4]
```diff
@@ -0,0 +1,353 @@
...
+            'Passwords shared with me'            : 'Met mij gedeelde wachtwoorden',
+            'Show Notifications for'              : 'Laat berichtgevingen zien voor',
+            'Other errors'                        : 'Andere fouten',
+            'Danger Zone'                         : 'Gevarenzone',
+            'Reset all settings'                  : 'Alle instellingen resetten',
+            'Reset'                               : 'Reset',
+            'Delete everything'                   : 'Alles verwijderen',
+            'DELETE EVERYTHING'                   : 'ALLES VERWIJDEREN',
+            'Account reset requested'             : 'Account reset aangevraagd',
+            'View'                                : 'Weergave',
```
[the remark this answers is not in the document]
Comment (roru69) [reply src/l10n/nl.js:274]: Added Default to the translation. It translates to " Standaard"  in dutch.

File: src/l10n/nl.json [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+        "HTTPS Required"                                                                            : "HTTPS verplicht",
+        "reload this page"                                                                          : "Pagina opnieuw laden",
+        "documentation"                                                                             : "Documentatie",
+        "This application requires %s in order to work safely."                                     : "Deze applicatie heeft %s nodig om veilig te functioneren.",
+        "Consult the %s to configure Nextcloud correctly."                                          : "Lees %s om Nextclout correct te configureren.",
+        "If you do not have a HTTPS certificate, get one for free from %s."                         : "Wanneer u geen HTTPS certificaat heeft kunt u deze kosteloos halen bij %s.",
+        "Enable %s to troubleshoot this error."                                                     : "Activeer %s om het probleem te onderzoeken.",
+        "You can try to %s the page with HTTPS."                                                    : "U kan proberen om %s de pagina met HTTPS.",
```
[the remark this answers is not in the document]
Comment (roru69) [reply src/l10n/nl.json]: Made another small change "deze" (this) changed to " de" (the).

File: src/l10n/nl.json [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+        "Saved"                                                                                     : "Opgeslagen",
+        "Cleared"                                                                                   : "Geleegd",
+        "Failed"                                                                                    : "Mislukt",
+        "Need help"                                                                                 : "Heeft u hulp nodig",
+        "found a bug"                                                                               : "Heeft u een bug gevonden",
+        "looking for the documentation"                                                             : "Zoekt u naar de documentatie",
+        "%s, %s or %s? We've got you covered!"                                                      : "%s, %s of %s? We staan voor u klaar!",
+        "PHP %s is no longer supported."                                                            : "PHP %s wordt niet meer ondersteund.",
```
[the remark this answers is not in the document]
Comment (roru69) [reply src/l10n/nl.json]: Made the change, added the new line.

File: src/l10n/nl.json [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+        "Clear this cache"                                                                          : "Deze cache legen",
+        "You can not clear a cache using a shared service"                                          : "U kan deze cache niet legen omdat deze gebuikt wordt door een gedeelde dienst",
+        "Saved"                                                                                     : "Opgeslagen",
+        "Cleared"                                                                                   : "Geleegd",
+        "Failed"                                                                                    : "Mislukt",
+        "Need help"                                                                                 : "Heeft u hulp nodig",
+        "found a bug"                                                                               : "Heeft u een bug gevonden",
+        "looking for the documentation"                                                             : "Zoekt u naar de documentatie",
+        "%s, %s or %s? We've got you covered!"                                                      : "%s, %s of %s? We staan voor u klaar!",
```
[the remark this answers is not in the document]
Comment (roru69) [reply src/l10n/nl.json]: "Heeft u hulp nodig, Heeft u een bug gevonden of Zoekt u naar de documentatie? We staan voor u klaar!" changed to "Heeft u hulp nodig, heeft u een bug gevonden of zoekt u naar de documentatie? We staan voor u klaar! "

File: src/l10n/nl.json [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+        "Enable %s to troubleshoot this error."                                                     : "Activeer %s om het probleem te onderzoeken.",
+        "You can try to %s the page with HTTPS."                                                    : "U kan proberen om %s de pagina met HTTPS.",
+        "If this problem persists, you should contact your administrator."                          : "Neem contact op met uw beheerder wanneer het probleem zich weer voordoet.",
+        "Passwords WebUI Access (see Passwords F.A.Q)"                                              : "Wachtwoorden WebUi toegang (Zie Wachtwoorden F.A.Q)",
+        "Open documentation"                                                                        : "Documentatie openen",
+        "Legacy Api Support"                                                                        : "Legacy API ondersteuning",
+        "Enable Legacy API"                                                                         : "Legacy AIP aanzetten",
```
[the remark this answers is not in the document]
Comment (roru69) [reply src/l10n/nl.json]: Indeed. Changed to API

File: src/l10n/nl.js (hunk 1 above)
Comment (roru69) [reply src/l10n/nl.js]: The translation in Dutch for "Quote" and "Double Quote"  is the same. "Hochkomma" is 
German, changed it to " Aanhalingsteken".

File: src/l10n/nl.js (hunk 2 above)
Comment (roru69) [reply src/l10n/nl.js]: Indeed, changed al the instances of " aktie" to "actie" (also the misspelled one "Atie". It seems that writing it with a " c"  is the preferred way.

**Review by marius-wieschollek — APPROVED** [review]
Looks good to me

**Pull request merged**